dora/src/bytecode/generator.rs

```diff
@@ -687,6 +699,7 @@ impl<'a, 'ast> AstBytecodeGen<'a, 'ast> {
                 Intrinsic::IntNeg => {
                     let dest = self.ensure_register(dest, BytecodeType::Int);
                     let src = self.visit_expr(&expr.opnd, DataDest::Alloc);
+                    self.gen.set_position(expr.pos);
```

Comment:
Do we really need a position for neg_int/neg_long/bool_not?

Reply:
Generally speaking I would say no. But if we don't set it, we get a incorrect position. As long as we track the position of every offset we should try to get it right.

Reply:
I removed some of the calls to `set_position`.